src/collection-preferences/__tests__/sticky-columns.test.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { CollectionPreferencesProps } from '../../../lib/components/collection-preferences';
+import { CollectionPreferencesWrapper } from '../../../lib/components/test-utils/dom';
+import { renderCollectionPreferences, stickyColumnsPreference } from './shared';
+
+function renderWithStickyColumnsPreferences(props: Partial<CollectionPreferencesProps>): CollectionPreferencesWrapper {
+  return renderCollectionPreferences({ stickyColumnsPreference, ...props });
+}
+
+const firstColumnOptions = [0, 1, 2];
+const lastColumnOptions = [0, 1];
+
+function isSelectedOption(
+  wrapper: CollectionPreferencesWrapper,
+  firstOrLast: 'first' | 'last',
+  expectedOption: number
+) {
+  const options = firstOrLast === 'first' ? firstColumnOptions : lastColumnOptions;
+  const radioGroupWrapper = wrapper.findModal()!.findStickyColumnsPreference()!.findRadioGroup(firstOrLast);
+  options.forEach(option => {
+    if (option === expectedOption) {
+      expect(radioGroupWrapper.findInputByValue(`${option}`)!.getElement()).toBeChecked();
+    } else {
+      expect(radioGroupWrapper.findInputByValue(`${option}`)!.getElement()).not.toBeChecked();
+    }
+  });
+}
+
+describe('Sticky columns preference selection', () => {
+  test('correctly displays titles', () => {
+    const wrapper = renderWithStickyColumnsPreferences({});
+    wrapper.findTriggerButton().click();
+    const preferencesModalWrapper = wrapper.findModal();
+
+    expect(preferencesModalWrapper!.findStickyColumnsPreference()!.findTitle('first').getElement()).toHaveTextContent(
+      'Stick first column(s)'
+    );
+    expect(preferencesModalWrapper!.findStickyColumnsPreference()!.findTitle('last').getElement()).toHaveTextContent(
```

Comment:
Shouldn't these be `preferencesModalWrapper!.findStickyColumnsPreference('first')!.findTitle()`instead?